src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Route, Routes } from 'react-router-dom'
+import './App.css'
+import ApplicantMain from './components/applicantMain/ApplicantMain.tsx'
+import EmployerMain from './components/employerMain/EmployerMain.tsx'
+import Home from './components/home/Home.tsx'
+import { ResumePage } from './components/resumePage/ResumePage.tsx'
+import ResumeSearch from './components/resumeSearch/ResumeSearch.tsx'
+import SignInPage from './components/signInPage/SignInPage.tsx'
+import SignUpPage from './components/signUpPage/SignUpPage.tsx'
+import { VacancyPage } from './components/vacancyPage/VacancyPage.tsx'
```

Comment:
looks unclear why ResumePage and VacancyPage imports as { something }, but all other pages imports as defaults. let's make all imports similar to each other